app/code/Magento/MediaGalleryRenditions/Model/Config.php

```diff
@@ -54,6 +55,16 @@ public function isEnabled(): bool
         return $this->scopeConfig->isSetFlag(self::XML_PATH_ENABLED);
     }
 
+    /**
+     * Check if the media gallery renditions is enabled
+     *
+     * @return bool
+     */
+    public function isMediaGalleryRenditionsEnabled(): bool
```

Comment:
New method introduction is not necessary, `isEnabled` method should be now based on `system/media_gallery_renditions/enabled` value